components_keys.py

```diff
@@ -0,0 +1,4 @@
+"""Store streamlit component keys"""
+
+class ComponentsKeys:
```

Comment:
qq. what's this use for?

Reply:
I added this to track the Streamlit component keys (especially if we add more components down the road). The idea behind this is when we add a file via the file manager instead of the `file_uploader`, we need to pass a component key. We need to do this for zip files since each unzipped file is not of type `UploadedFile` (the type returned from `file_uploader`). In order for the zip file to play nice with the rest of the code, I converted each of its unzipped files to `UploadedFile` so we don't have to do any hacky checks to the file type.

I've never used Streamlit before so maybe there's a better way.